server/src/main/java/com/defold/extender/remote/RemoteEngineBuilder.java

```diff
@@ -188,20 +189,20 @@ public void buildAsync(final RemoteInstanceConfig remoteInstanceConfig,
                     File targetResult = new File(resultDir, BuilderConstants.BUILD_RESULT_FILENAME);
                     Files.move(tmpResult.toPath(), targetResult.toPath(), StandardCopyOption.ATOMIC_MOVE);
                 } else {
-                    String errorLog = EntityUtils.toString(response.getEntity());
-                    LOGGER.error(Markers.COMPILATION_ERROR, String.format("Failed to build source.\n%s", errorLog));
+                    LOGGER.error(Markers.COMPILATION_ERROR, "Failed to build source.");
```

Comment:
I think we don't want to see full build log in logging system. It should be enough to see error at the last step which should be logged in the moment of fail. And than we can collect full work log by traceId.